kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
We can add ``--env HOST="`hostname -s`"`` when running the docker and leverage the `$HOST` variable inside the container. Not sure how though, perhaps docker could have a special start script that will do some sed `s;http://localhost;$HOST;` on the realm.json and then run the WF.